exec/java-exec/src/main/resources/rest/profile/profile.ftl

```diff
@@ -372,6 +372,16 @@ table.sortable thead .sorting_desc { background-image: url("/static/img/black-de
   </div> <br>
 
     <script>
+    //Inject Spilled Tags
+    $(window).on('load', function () {
+      var spillLabel = document.getElementsByClassName("spill-tag");
+      var i;
+      for (i = 0; i < spillLabel.length; i++) {
+        var content = spillLabel[i].innerHTML;
+        spillLabel[i].innerHTML = "<span class=\"glyphicon glyphicon-download-alt\">&nbsp;</span>"+content;
+      }
+    });
+
```

Comment:
Why do you need exec/java-exec/src/main/resources/rest/static/img/popout.png?

Reply:
It was actually meant as a part of DRILL-6289. Will remove it.